phase0/CRC Design/GUIButtonLoad.txt

```diff
@@ -0,0 +1,10 @@
+Class name: GUIButtonLoad
+Parent class: GUIButton
+Subclasses: 
+
+Responsibilities:
+* Loads from an RDF file using the Load class when clicked
```

Comment:
I think that it connects to the Load class, but it doesn't do the conversion itself. It sends some Game or State instance into Load, and Load handles everything.